lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -1,206 +1,328 @@
-.search-page {
-    background-color: $color-bg-gray;
+// stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type
+body {
+    background-color: $color-bg-gray-blue-light;
 }
 
-.heading-wrapper {
-    text-align: center;
+.heading-wrapper-mobile {
+    background-image: url('images/default-brand/bg-dark.jpg');
+    background-color: $osf-dark-blue-navbar;
     color: $color-text-white;
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+
+    label > h1 {
+        font-weight: 500;
+        margin: 0.2rem;
+        padding: 0.75rem 0 0.5rem 0.5rem;
+    }
+
+    span {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin-left: 0;
+        padding: 0 0 0.5rem 0.5rem;
+        width: 100%;
+
+        .help-button-mobile {
+            transform: scale(0.5);
+            border-radius: 50%;
+            margin-left: 0;
+            position: absolute;
+            right: 0.1rem;
+        }
+    }
+
+    .sidenav-toggle {
+        width: 100%;
+    }
+}
+
+.left-panel-mobile {
+    padding: 1rem;
+    width: 75vw;
+
+    .object-type-dropdown {
+        label > div {
+            width: 195px;
+        }
+    }
+
+    .object-sort-dropdown {
+        margin-bottom: 1rem;
+
+        label > div {
+            width: 193px;
+        }
+    }
+}
+
+.search-main-mobile {
+    > div {
+        margin: 25px auto;
+        padding: 20px;
+        width: 90vw;
+    }
+
+    .pagination-buttons {
+        padding-right: 0;
+    }
+}
+
+.search-page > div:first-of-type {
+    max-width: 95vw;
+    margin-left: 5rem;
+}
```

Comment:
This rule seems to be targeting the main container of osf-layout. I don't think we want this style, as this overwrites the default behavior of osf-layout, and can also lead to large margins that is not appropriate on smaller screen sizes
![image](https://github.com/CenterForOpenScience/ember-osf-web/assets/51409893/e5c4cd5a-a6bb-43f0-9397-e904001f2e23)

Reply:
This most accurately matches the mock-ups we were given:

<img width="1201" alt="image" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/4bf6993d-392e-4453-882f-d67789934d76">

Mobile, tablet, and the in-between sizes are now accounted for as well. If we wanted a deviation from the mock-ups, that would have been a conversation for a prior meeting.

Reply:
Please see the other comment related to this. I'm happy to discuss it further if you like.